pkg/antctl/command_message.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package antctl
+
+import (
+	"fmt"
+	"k8s.io/client-go/rest"
+	"net/http"
+)
+
+func generateMessage(opt *requestOption, result rest.Result) error {
+	var code int
+	_ = result.StatusCode(&code)
+	if err := generate(opt.commandDefinition, opt.args, code); err != nil {
+		return err
```

Comment:
the logic is fine, but it's a bit weird that 'err' is not an error code but the actual return value of the function! Perhaps errMsg could be a better name. (I know this is a bit pedant, sorry...)

Reply:
It makes sense. Code updated.